pex/bin/pex.py

```diff
@@ -159,7 +159,7 @@ def configure_clp_pex_resolution(parser, builder):
   group.add_option(
       '--cache-dir',
       dest='cache_dir',
-      default=os.path.expanduser('~/.pex/build'),
+      default=os.path.expanduser('{pex_root}/build'),
```

Comment:
should this lose the `expanduser` treatment now that we don't specify a string literal with ~ in it?